pkg/apis/componentconfig/types.go

```diff
@@ -158,13 +160,13 @@ type KubeletConfiguration struct {
 	AllowPrivileged bool `json:"allowPrivileged"`
 	// hostNetworkSources is a comma-separated list of sources from which the
 	// Kubelet allows pods to use of host network. Defaults to "*".
```

Comment:
Are the constants at `k8s.io/kubernetes/pkg/kubelet/types/pod_update.go` exhaustive or are there any others we should list?